server/playground.py

```diff
@@ -16,13 +17,21 @@
 
 app = Flask(__name__)
 
+# CORS
+@app.after_request
+def after_request(response):
+    response.headers.add('Access-Control-Allow-Origin', '*')
+    response.headers.add('Access-Control-Allow-Headers', 'Content-Type,Authorization')
+    response.headers.add('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE')
+    return response
+
 
 @app.route('/', methods=['GET', 'POST'])
 def handle_request():
     if request.method == 'GET':
         return json.dumps(get_available_domains())
     elif request.method == 'POST':
-        return execute_pipeline(request.get_json())
+        return execute_pipeline(request.get_json(), **request.args)
```

Comment:
I the dict > json conversion is automatic, we should be able to get rid of the `json.dumps` above, too ?

Reply:
ah yeah true! Flask handles it for us

Reply:
cb1f366

Reply:
Arf, actually no: `
TypeError: The view function did not return a valid response. The return type must be a string, dict, tuple, Response instance, or WSGI callable, but it was a list.
`

because this methods returns a list, not a dict